server/src/main/java/org/opentosca/toscana/plugins/kubernetes/docker/image/ImageBuilder.java

```diff
@@ -69,7 +69,7 @@ public void progress(ProgressMessage progressMessage) throws DockerException {
             log(progressMessage.progress(), logger::trace);
         log(progressMessage.stream(), logger::info);
         log(progressMessage.error(), logger::error);
-        log(progressMessage.status(), logger::info);
+        log(progressMessage.status(), logger::trace);
```

Comment:
Can this get set back to `info`?

Reply:
if you want approx. 200 Lines saying downloading/extracting/Checking for sure :)

Reply:
This is currently set to `trace`, not `info`.

Reply:
Hmm thats weird. Why is it producing less log output then?

Before: https://gist.github.com/c-mueller/031863766bfbe65b172243eb4bbb169f
After: https://gist.github.com/c-mueller/c1d5d034163b6b488c6b7785f4c47aac

Reply:
Maybe its a bug in the docker library.

Reply:
Where is the bug? i am changing the level from info to trace. Trace beeing even higher then debug.